### PR TITLE
replace method getNameOrId by getOptionalName

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**
Signed-off-by: walid-sahnoun <walid.sahnoun@rte-france.com>

### CODE REVIEW

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java [hunk 1]
```diff
@@ -482,7 +483,7 @@ private static VscConverterStationMapData toMapData(VscConverterStation vscConve
     private static LoadMapData toMapData(Load load) {
         Terminal terminal = load.getTerminal();
         LoadMapData.LoadMapDataBuilder builder = LoadMapData.builder()
-            .name(load.getNameOrId())
+            .name(load.getOptionalName().orElse(StringUtils.EMPTY))
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/network/map/NetworkMapService.java:486]: orElse(null) ?

This should be done for all equipments type, and not only for the load ...

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/network/map/NetworkMapService.java (hunk 1 above)
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/network/map/NetworkMapService.java]: done

**Review by FranckLecuyer — APPROVED** [review]

**Comment by SlimaneAmar** [comment]
Add a test with id and name not null and different

File: src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java [hunk 2]
```diff
@@ -493,6 +493,7 @@ public void setUp() {
                 .withSectionIndex(2)
                 .add();
 
+        vlnew2.newLoad().setId("LOAD_WITH_NULL_NAME").setBus("NNEW2").setConnectableBus("NNEW2").setP0(600.0).setQ0(200.0).setName(null).add();
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java:496]: Add a test with id and name not null and different

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java (hunk 2 above)
Comment (Walid-Sahnoun) [reply src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java:496]: i add a test with name not null
but what do you mean different ?

File: src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java [hunk 3]
```diff
@@ -493,6 +493,8 @@ public void setUp() {
...
+        vlnew2.newLoad().setId("LOAD_ID_NAME").setBus("NNEW2").setConnectableBus("NNEW2").setP0(600.0).setQ0(200.0).setName("LOAD_NAME").add();
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java:497]: Add new line

Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java:497]: Rename LOAD_ID_NAME by LOAD_ID

File: src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java (hunk 2 above)
Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java:496]: id and name different

File: src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java (hunk 3 above)
Comment (Walid-Sahnoun) [reply src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java]: ok

Comment (Walid-Sahnoun) [reply src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java]: done

**Review by SlimaneAmar — APPROVED** [review]
